tests/e2e/test_main.py

```diff
@@ -867,14 +882,16 @@ def _run_test_data_cp_from_cloud_to_local_compress(
     args_data_cp_from_cloud: Callable[..., List[str]],
     bucket: str,
     archive_extension: str,
+    use_tmp_dir: bool,
 ) -> None:
     TEMP_UNPACK_DIR.mkdir(parents=True, exist_ok=True)
     with TemporaryDirectory(dir=TEMP_UNPACK_DIR.expanduser()) as tmp_dir:
-        src = f"{bucket}/hello.{archive_extension}"
+        # if src and dst archive types are the same - compression is skipped.
+        src = f"{bucket}/hello.{random.choice(TESTED_ARCHIVE_TYPES)}"
```

Comment:
It's very bad to have randomness in tests. I'll fix it while fixing conflicts with https://github.com/neuro-inc/neuro-extras/pull/127/

Reply:
fixed

Reply:
note: yes we do generate random files in our tests and use them, but this doesn't imply the logic of the test, so it's acceptable, but also not very good

Reply:
added fix back in c555965